sycl/include/CL/__spirv/spirv_vars.hpp

```diff
@@ -15,7 +15,7 @@
 
 #define __SPIRV_VAR_QUALIFIERS extern "C" const
 
-#if defined(__SYCL_NVPTX__)
+#if defined(__SYCL_NVPTX__) || defined(__SYCL_AMDGCN__)
```

Comment:
(Since `__SYCL_NVTPX__` is being removed in https://github.com/intel/llvm/pull/3977)
```suggestion
#if defined(__SYCL_NVPTX__) || defined(__AMDCGN__)
```